### PR TITLE
Graphics/Shaders: Minor changes

### PR DESCRIPTION
**Pull request by lioncash**
Just a few minor changes that cull some includes and make a few arrays nicer to work with. Each commit should be cleanly separated for review.

---
This change is [Reviewable](https://reviewable.io/reviews/axiodl/urde/88)

### CODE REVIEW

File: Runtime/World/CFluidPlaneGPU.cpp [hunk 1]
```diff
@@ -52,19 +52,19 @@ void CFluidPlaneGPU::RenderStripWithRipples(float curY, const CFluidPlaneRender:
...
+      pv.m_innerLevels.fill(subdivF);
     } else {
-      bool r19 = (flags[yTile + 1][xTile] & 0x2) != 0; // North
-      bool r16 = (flags[yTile][xTile - 1] & 0x8) != 0; // West
-      bool r18 = (flags[yTile][xTile + 1] & 0x4) != 0; // East
-      bool r17 = (flags[yTile - 1][xTile] & 0x1) != 0; // South
+      const bool r19 = (flags[yTile + 1][xTile] & 0x2) != 0; // North
+      const bool r16 = (flags[yTile][xTile - 1] & 0x8) != 0; // West
+      const bool r18 = (flags[yTile][xTile + 1] & 0x4) != 0; // East
+      const bool r17 = (flags[yTile - 1][xTile] & 0x1) != 0; // South
```
Comment (Antidote) [inline Runtime/World/CFluidPlaneGPU.cpp]: Probably be a good idea to go ahead and rename these `r#` variables something meaningful

Comment (lioncash) [reply Runtime/World/CFluidPlaneGPU.cpp]: Done. I've put the changes in a separate commit to keep it separate from the std::array conversion changes.

Comment (Antidote) [reply Runtime/World/CFluidPlaneGPU.cpp]: LGTM

**Comment by Antidote** [comment]
@jackoalan please review this.

**Comment by Antidote** [comment]
This PR has a bunch of broken references to `EFilterType`
This is due to missing CCameraFilter.hpp inclusion is various places.

**Comment by Antidote** [comment]
@lioncash can you please fix the issue I mentioned?
I really want to go ahead and merge this.

**Comment by lioncash** [comment]
@lioncash Done, that should hopefully address all of them.

**Comment by Antidote** [comment]
Thanks! I'll pull and check before merging.

**Comment by Antidote** [comment]
Just missing one reference in `Runtime/AutoMapper/CMappableObject.cpp`

**Comment by lioncash** [comment]
Done

**Comment by Antidote** [comment]
Thanks! Everything looks good, so I went ahead and merged.

**Pull request merged**